skin/frontend/base/default/ebanx/js/document.js

```diff
@@ -47,7 +47,10 @@ var changeTaxVatLabel = function () {
     var taxVatMask = newLabel.indexOf('CNPJ') !== -1 ? ['999.999.999-99', '99.999.999/9999-99'] : ['999.999.999-99', '999.999.999-99'];
     VMasker(taxVatInput).maskPattern(taxVatMask[0]);
     taxVatInput.addEventListener('input', inputHandler.bind(undefined, taxVatMask, 14), false);
+    return true;
```

Comment:
Why are you doing this? Change the order!!! Always unmask the field, then check if you need to mask it again. Also, since you are making changes there, put `country === 'BR' && taxVatInput` on a variable, like `shouldMask` and use it like:
```javascript
VMasker(taxVatInput).unMask();

var shouldMask = country === 'BR' && taxVatInput;
if (shouldMask) {
  VMasker(taxVatInput).maskPattern(taxVatMask[0]);
  taxVatInput.addEventListener('input', inputHandler.bind(undefined, taxVatMask, 14), false);
}
```
Also, why returning `true`?